### PR TITLE
Fix last Ubuntu 22.04 test case issues @open sesame 8/3 08:53

### PR DESCRIPTION
**Pull request by myungjoo**
The following two commits are included:

commit 605ba3bdbee75528f917dc40267186ccbb5f89d8 (HEAD -> fix/python310)
Author: MyungJoo Ham <myungjoo.ham@samsung.com>
Date:   Tue Aug 2 14:49:29 2022 +0900

    join/test-case: remove undeterminism.
    
    Because of the undeterministic nature of parallel streams,
    the unittest case results sometime vary.
    Specify the synchronization policy so that the results
    become deterministic.
    
    Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

commit 7f952430cba64ca4ccbd5a82af11b85debccc31f
Author: MyungJoo Ham <myungjoo.ham@samsung.com>
Date:   Tue Aug 2 13:49:23 2022 +0900

    python: fix python 3.10 PyObject type errors.
    
    From Python 3.10 python->C/C++ object transfer
    has stricter type checks. Floats are no more
    supported by PyLong.
    
    Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

**Review by jaeyun-jung — APPROVED** [review]

File: tests/gstreamer_join/runTest.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} \
         videotestsrc num-buffers=4 pattern=12 ! videoconvert ! videoscale ! video/x-raw,format=RGB,width=160,height=120 ! tensor_converter ! mux.sink_0 \
         videotestsrc pattern=13 ! videoconvert ! videoscale ! video/x-raw,format=RGB,width=160,height=120 ! tensor_converter ! mux.sink_1 \
         videotestsrc pattern=15 ! videoconvert ! videoscale ! video/x-raw,format=RGB,width=160,height=120 ! tensor_converter ! mux.sink_2 \
-        tensor_mux name=mux ! tensor_if name=tif compared-value=TENSOR_AVERAGE_VALUE compared-value-option=0 supplied-value=100 operator=LT then=TENSORPICK then-option=1 else=TENSORPICK else-option=2 \
```
[the remark this answers is not in the document]
Comment (myungjoo) [reply tests/gstreamer_join/runTest.sh:33]: this "Join" issue appears to be "occasional" and "not related with 22.04". I'll remove this from this PR.

[the remark this answers is not in the document]
Comment (myungjoo) [reply tests/gstreamer_join/runTest.sh]: removed.

**Review by anyj0527 — APPROVED** [review]
LGTM ✔

**Review by songgot — APPROVED** [review]
LGTM

**Pull request merged**